cmd/uzfs_test/uzfs_txg_diff.c

```diff
@@ -166,16 +186,37 @@ uzfs_zvol_txg_diff_blk_test(void *arg)
 
 	while (i++ < test_iterations) {
 		count = 0;
+		max_io = MAX(uzfs_random(100), 5);
 
 		txg_wait_synced(spa_get_dsl(spa), 0);
 		first_txg  = spa_last_synced_txg(spa);
 
-		while (count++ < diff_txg) {
+		while (count++ < max_io) {
 			io_num++;
 			blk_offset = uzfs_random(vol_blocks - 16);
+			/*
+			 * make sure offset is aligned to block size
+			 */
 			offset = ((blk_offset * blksz + block_size) /
```

Comment:
offset can be just blk_offset * block_size

Reply:
also, have a test case with less active size (probably for 30K or 50K blocks), so that, collisions/merges of tree can be verified